internal/flags/configure.go

```diff
@@ -237,6 +241,103 @@ func (f *Flags) handleConfigureTLS() error {
 	return nil
 }
 
+func (f *Flags) handleAddEthernetSettings() error {
+	var configJson string
+	fs := f.NewConfigureFlagSet(utils.SubCommandAddEthernetSettings)
+	fs.StringVar(&f.configContent, "config", "", "specify a config file or smb: file share URL")
+	fs.StringVar(&configJson, "configJson", "", "configuration as a JSON string")
+	fs.BoolVar(&f.IpConfiguration.DHCP, "dhcp", false, "dhcp usage")
+	fs.BoolVar(&f.IpConfiguration.StaticIp, "staticip", false, "staticip usage")
```

Comment:
Talked to @matt-primrose about it, its a nit-pick , but can we change `-staticip` to `-static`